data/filter_options.py

```diff
@@ -32,7 +32,7 @@ def _validate_type(self, check, value, attribute):
       raise ValueError(error)
 
   def _is_valid_state(self, state):
-    return 0 <= state <= 4
+    return 0 <= state <= 6
```

Comment:
Since you change from 1 to 6 you have to change to 1 <= state <= 6

Reply:
Maybe move this to the State class class method, and call State.is_valid(int), such that you can wherever you use State.

Reply:
Done